_security/configuration/index.md

```diff
@@ -13,19 +13,102 @@ redirect_from:
+
+   ```
+   plugins.security.ssl.transport.pemcert_filepath: /path/to/your/cert.pem
+   plugins.security.ssl.transport.pemkey_filepath: /path/to/your/key.pem
+   plugins.security.ssl.transport.pemtrustedcas_filepath: /path/to/your/ca.pem
+   plugins.security.ssl.http.enabled: true
+   plugins.security.ssl.http.pemcert_filepath: /path/to/your/cert.pem
+   plugins.security.ssl.http.pemkey_filepath: /path/to/your/key.pem
+   plugins.security.ssl.http.pemtrustedcas_filepath: /path/to/your/ca.pem
+   ```
+
+### [Reconfigure `config.yml` to use your authentication backend]({{site.url}}{{site.baseurl}}/security/configuration/configuration/)
+
+The `config.yml` file allows you to configure the authentication and authorization mechanisms for OpenSearch. Update the authentication backend settings in `config/opensearch-security/config.yml` according to your requirements. For example, to use LDAP as your authentication backend, add the following settings:
+
+  ```
+    authc:
+      basic_internal_auth:
+        http_enabled: true
+        transport_enabled: true
+        order: 1
+        http_authenticator:
+          type: basic
+          challenge: true
+        authentication_backend:
+          type: internal
+   ```
+
+### [Modify the configuration YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml/).
+
+Determine if any additional YAML files need modification, for example the `roles.yml`, `roles_mapping.yml`, `internal_users.yml`. Edit the files with any additional configuration needed.
+
+### [Set a password policy]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#password-settings/)
+
+When using the internal user database, we recommend enforcing a password policy to make sure that strong passwords are used. For information strong password policies, see [set a password policy in `opensearch.yml`]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#opensearchyml).
+
+### [Apply changes using the `securityadmin` script]({{site.url}}{{site.baseurl}}/security/configuration/security-admin/)
+
+The following steps are not required for first-time user because the security index is automatically initialized from the YAML configuration files when OpenSearch starts.
+{: .note}
+
+After initial setup, if you make changes to your security configuration or if you disable automatic initialization by setting `plugins.security.allow_default_init_securityindex` to `false` (setting to prevent security index initializing from `yaml` files), you need to manually apply changes using the `securityadmin` script.
+
+1. Find the `securityadmin` script. The script is typically found in the OpenSearch plugins directory, `plugins/opensearch-security/tools/securityadmin.sh`. If you're using OpenSearch 1.x, the `securityadmin` script was located in the `plugins/opendistro_security/tools/` directory. 
+2. Run the script with the appropriate parameters by using the following command:
```

Comment:
@cwperks: How would I know what the appropriate parameters are?

Reply:
The appropriate parameters are given in the example that immediately follows. I have made this clearer by making the example clearer (by giving it in the same format as is on the other documentation page: [Applying changes to configuration files](https://opensearch.org/docs/latest/security/configuration/security-admin/)). 

And also by providing links to other relevant pages with more information: [Applying changes to configuration files](https://opensearch.org/docs/latest/security/configuration/security-admin/) & [securityadmin.sh Troubleshooting](https://opensearch.org/docs/latest/troubleshoot/security-admin/)

Thanks Nate!

Reply:
@Naarcha-AWS You can find the commands in the Basic Usage section here: https://opensearch.org/docs/latest/security/configuration/security-admin/#basic-usage